message/aws_sled.py

```diff
@@ -21,3 +21,10 @@ def s3():
             verify=False
         )
     return boto3.resource('s3')
+
+def stepFn():
+  region = os.getenv('AWS_DEFAULT_REGION', 'us-east-1')
+  if ('ENV' in os.environ) and (os.environ["ENV"] == 'testing'):
+    return boto3.client(service_name='stepfunctions', endpoint_url=localhost_s3_url, region_name=region)
```

Comment:
I'm not sure if step function is supported by localstack. Did you test this @abarciauskas-bgse ?

Reply:
It's not supported, I should put a comment here as well but this is just so we don't make real requests to AWS API in https://github.com/cumulus-nasa/cumulus-sled/blob/f259889965b9b013172806ab81b201b50a39ce01/examples/example-node-sled-lib.js

Reply:
Added comment in https://github.com/cumulus-nasa/cumulus-sled/pull/2/commits/40439dbf7a30b69fce504c0cc2644f8cefaaf07a